docs_outdoornav_user_manual/customized_tuning/tuning_parameters/navigation_parameters.mdx

```diff
@@ -39,117 +42,126 @@ The MPC controller operates as a state machine that contains the following state
+| `goal_tolerance_xy_nortk` | The (x, y) tolerance value (ie. radius) assigned to a goal, when RTK is disabled, in order to determine whether or not to consider the goal complete. | [/navigation/*&ltcontroller&gt*/path_tracker](#file_location) | **m** |
+| `goal_tolerance_yaw_nortk` | The yaw tolerance value (ie. heading) assigned to a goal, when RTK is disabled, in order to determine whether or not to consider the goal complete. | [/navigation/*&ltcontroller&gt*/path_tracker](#file_location) | **rad** |
+
+#### Corner State Parameters {#corner_state_params}
+
+The following parameters can be modified to tune the behavior of the controller during and as it
+as it approaches corners.
 
 | Parameter | Description | Namespace | SI Units |
 |-----------|-------------|-----------|:--------:|
-| `corner_segment_slowdown` | Flag to enable/disable the slowdown behaviour in MPC state: CORNER | navigation/*&ltcontroller&gt*/path_tracker| **bool** |
-| `corner_lookahead` | The distance on the reference path from the platforms current location, along which to determine if a corner is present. If a corner is present the MPC state will switch to CORNER state and begin reducing the horizon accordingly. | navigation/*&ltcontroller&gt*/path_tracker| **m** |
-| `corner_detection_threshold` | Theshold on the path curvature, between the platforms current location and the end of the corner_lookahead distance, above which to switch the MPC in state: CORNER, and begin reducing the horizon accordingly. | navigation/*&ltcontroller&gt*/path_tracker| **rad** |
-| `corner_slowdown_multiplier` | The multiplier by which to increase the MPC horizon during the MPCs CORNER state in order to slow the platform down from its top speed. This value should be greater than 1.0. | navigation/*&ltcontroller&gt*/path_tracker| **--** |
-| `endpoint_multiplier_corner` | Weight multiplier of the very last point along the local plan segment in MPC state: CORNER. This value ensures convergence/stability of the optimizer by mimicking the MPCs infinite horizon by applying a terminal cost. | navigation/*&ltcontroller&gt*/path_tracker| **--** |
+| `corner_segment_slowdown` | Flag to enable/disable the slowdown behaviour in MPC state: CORNER | [/navigation/*&ltcontroller&gt*/path_tracker](#file_location)| **bool** |
+| `corner_lookahead` | The distance on the reference path from the platforms current location, along which to determine if a corner is present. If a corner is present the MPC state will switch to CORNER state and begin reducing the horizon accordingly. | [/navigation/*&ltcontroller&gt*/path_tracker](#file_location)| **m** |
+| `corner_detection_threshold` | Threshold on the path curvature, between the platforms current location and the end of the corner_lookahead distance, above which to switch the MPC in state: CORNER, and begin reducing the horizon accordingly. | [/navigation/*&ltcontroller&gt*/path_tracker](#file_location)| **rad** |

```

Comment:
platforms current location -> Should this maybe be UGV's current location. I've seen platform used in other cases where it's more discussing the actual software side and I think it's appropriate there but I wonder if in this case it's more of about the physical robot.